modules/op-release-notes-1-15.adoc

```diff
@@ -472,3 +472,19 @@ With this update, {pipelines-title} General Availability (GA) 1.15.1 is availabl
 * Before this update, when you enabled the {pipelines-shortname} plugin, viewed the details page for a pipeline in the web console, and selected *Edit Pipeline* from the menu, the console displayed the YAML specification of the pipeline. With this update, the console displays the *Pipeline Builder* page.
 
 * Before this update, in {pipelines-shortname} version 1.15.0, when you added a comment on a pull request, {pac} set an event type depending on the comment content, for example, `retest-comment` or `on-comment`. With this update, the event type after a pull request comment is always `pull_request`, similar to {pipelines-shortname} version 1.14 and earlier.
+
+[id="release-notes-1-15-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.15.2
+
+With this update, {pipelines-title} General Availability (GA) 1.15.2 is available on {OCP} 4.14 and later versions.
+
+[id="fixed-issues-1-15-2_{context}"]
+=== Fixed issues
+
+* Before this update, when you passed a parameter value to a pipeline or task and the value included more than one variable with both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly. The pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
+
+* Before this update, in the web console, sorting of pipeline runs and task runs by creation time did not work. Sorting of pipelines by last run time also did not work. With this update, the sorting works correctly.
+
+* Before this update, in the web console, if you enabled the {pipelines-shortname} console plugin, when starting a pipeline you could not select the storage class for the volume claim template, because the *StorageClass* list was not present in the *VolumeClaimTemplate* options. With this update, you can select the storage class for the volume claim template.
+
+* Before this update, if you used {pipelines-as-code}, in the *Repository details* page of the web console the list of pipeline runs did not display correctly. With this update, the list of pipeline runs displays correctly.
```

Comment:
I would remove this second comma, or shift it to later in the sentence:
```suggestion
* Before this update, if you used {pipelines-as-code} in the *Repository details* page of the web console, the list of pipeline runs did not display correctly. With this update, the list of pipeline runs displays correctly.
```

Reply:
Unfortunately removing or moving the second comma makes the sentence incorrect. Using PAC is something you do in general, and not in the web console (usually). But if you do use PAC, then in the web console the list does not display correctly.

Reply:
I see what you mean, now that I look at this sentence again it makes more sense. Feel free to keep it as-is, but one other suggestion might be to switch around part of this sentence (assuming it remains accurate):
`Before this update, if you used {pipelines-as-code}, the list of pipeline runs did not display correctly in the *Repository details* page of the web console.`

But it's up to you, probably fine either way.